Mathlib/Algebra/Order/CauSeq/Completion.lean

```diff
@@ -276,6 +276,7 @@ noncomputable instance Cauchy.divisionRing : DivisionRing (Cauchy abv) where
   mul_inv_cancel x := CauSeq.Completion.mul_inv_cancel
   ratCast q := ofRat q
   ratCast_mk n d hd hnd := by rw [← ofRat_ratCast, Rat.cast_mk', ofRat_mul, ofRat_inv]; rfl
+  qsmul := qsmulRec _ -- TODO: fix instance diamond
```

Comment:
Maybe make a separate issue or add to an existing one.

Reply:
Easier to just fix it, #11263